content/integrations/google_cloud_bigtable/Java/_index.md

```diff
@@ -0,0 +1,377 @@
+---
+title: "Java"
+date: 2018-07-30T18:44:00-07:00
+draft: false
+weight: 3
+class: "resized-logo"
+---
+
+![](/images/java.png)
+
+For more information, you can read about it here and get started [Bigtable docs](https://cloud.google.com/bigtable/docs).
+
+{{% notice note %}}
+This guide makes use of a couple of APIs
+
+API|Reference
+---|---
+Bigtable how-to-guides|[Google Cloud Platform Bigtable how-to-guides](https://cloud.google.com/bigtable/docs/how-to/)
+Stackdriver|[Stackdriver codelab](/codelabs/stackdriver/)
+{{% /notice %}}
+
+Cloud Bigtable (cbt) is a petabyte-scale, fully managed NoSQL database service for large analytical and operational workloads.
+For more information you can read about it here and get started [Bigtable docs](https://godoc.org/cloud.google.com/go/bigtable/)
```

Comment:
nit: Update the url to https://googlecloudplatform.github.io/google-cloud-java/google-cloud-clients/apidocs/index.html.

Reply:
Actually, that section is on the language agnostic section of Bigtable so I've updated it to https://cloud.google.com/bigtable/docs/how-to